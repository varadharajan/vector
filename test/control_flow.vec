int vec_main()
{
    i := 0;

    while (i < 10) {
        if (i % 2 == 0)
            printf("%d\n", i);
        i++;
    }

    for (i in 0:5:2, j in 0:2)
        printf("%d %d\n", i, j);

    return 0;
}
