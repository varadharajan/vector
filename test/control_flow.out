0
2
4
6
8
0 0
0 1
2 0
2 1
4 0
4 1
